pkg/util/slice/slice.go

```diff
@@ -68,3 +68,16 @@ func ContainsString(slice []string, s string, modifier func(s string) string) bo
 	}
 	return false
 }
+
+// Deduplicate remove the duplicate value in the slice and keep the order unchanged
+func Deduplicate(s []string) (ret []string) {
```

Comment:
We usually try to avoid named return variables unless there would be confusion otherwise.